### PR TITLE
jupyter setup can get away with binary prereqs instead of source prereqs for drake

### PR DESCRIPTION
**Pull request by RussTedrake**
This change is [Reviewable](https://reviewable.io/reviews/russtedrake/underactuated/316)

### CODE REVIEW

**Comment by RussTedrake** [comment]
Currently blocked by https://github.com/RobotLocomotion/drake/issues/12793

**Comment by RussTedrake** [comment]
(i'm not sure why this doesn't also cause failures on underactuated bionic CI).

File: scripts/setup/jupyter_setup.py [hunk 1]
```diff
@@ -17,11 +17,11 @@ def setup_drake():
...
+                u"if [ ! -d '/opt/drake' ]; then curl -o drake.tar.gz https://drake-packages.csail.mit.edu/drake/continuous/drake-latest-mac.tar.gz && tar -xzf drake.tar.gz -C /opt && export HOMEBREW_CURL_RETRIES=4 && brew bundle --file=/opt/drake/share/drake/setup/Brewfile --no-lock; fi"  # noqa
             )
         elif platform.linux_distribution() == ("Ubuntu", "18.04", "bionic"):
             get_ipython().system(
-                u"if [ ! -d '/opt/drake' ]; then curl -o drake.tar.gz https://drake-packages.csail.mit.edu/drake/continuous/drake-latest-bionic.tar.gz && tar -xzf drake.tar.gz -C /opt && /opt/drake/share/drake/setup/install_prereqs; fi"  # noqa
+                u"if [ ! -d '/opt/drake' ]; then curl -o drake.tar.gz https://drake-packages.csail.mit.edu/drake/continuous/drake-latest-bionic.tar.gz && tar -xzf drake.tar.gz -C /opt && apt-get install -o APT::Acquire::Retries=4 -o Dpkg::Use-Pty=0 -qy --no-install-recommends $(cat /opt/drake/share/drake/setup/packages-bionic.txt); fi"  # noqa
```
Comment (jamiesnape) [inline scripts/setup/jupyter_setup.py]:
```
if [ ! -d '/opt/drake' ]; then curl -o drake.tar.gz https://drake-packages.csail.mit.edu/drake/continuous/drake-latest-bionic.tar.gz && tar -xzf drake.tar.gz -C /opt && apt-get update -o APT::Acquire::Retries=4 -qq && apt-get install -o APT::Acquire::Retries=4 -o Dpkg::Use-Pty=0 -qy --no-install-recommends $(cat /opt/drake/share/drake/setup/packages-bionic.txt); fi
```

Comment (RussTedrake) [reply scripts/setup/jupyter_setup.py]: that fixed it.  thanks!

**Pull request merged**